src/__swaps__/utils/swaps.ts

```diff
@@ -260,42 +260,55 @@ export function valueBasedDecimalFormatter({
 }): string {
   'worklet';
 
-  function calculateDecimalPlaces(): number {
-    const fallbackDecimalPlaces = MAXIMUM_SIGNIFICANT_DECIMALS;
+  function calculateDecimalPlaces(): {
+    minimumDecimalPlaces: number;
```

Comment:
`calculateDecimalPlaces` now returns `minimumDecimalPlaces` and `maximumDecimalPlaces` and will serve as a foundation for future decimal formatting improvements.